pkg/services/govmomi/vcenter/clone.go

```diff
@@ -271,8 +271,12 @@ func Clone(ctx *context.VMContext, bootstrapData []byte, format bootstrapv1.Form
 		// select one of the datastores of the owning cluster of the resource pool that matched the
 		// requirements of the storage policy.
 		if datastoreRef == nil {
-			r := rand.New(rand.NewSource(time.Now().UnixNano())) //nolint:gosec
```

Comment:
I think it's fine to keep using math/rand here - but let's add a comment explaining that we don't need any guarantee of pseudorandomness here